config/settings/test.yml

```diff
@@ -12,3 +12,4 @@ feature_flags:
   bibliography_resource: true
   test-flag-exhibit-slug:
     test_thing: true
+    index_related_content: true
```

Comment:
Can this just blindly be set to `true` in the test suite?

Reply:
I think so. The idea here was to only enable for this one exhibit to test the check on it. Should Am I missing another case here?

Reply:
Nah, I think I get what you're saying.  Because it'll ultimately be turned on specifically for parker (initially?) we want to test that the exhibit-specific configuration is what is working?  I didn't catch that this was tested in a way that validated that the feature would only be present in the exhibit it is specifically configured to.

Was just more of a passing question than anything, so 👍